integration-libs/s4-service/root/s4-service-root.module.ts

```diff
@@ -34,15 +37,66 @@ export function defaultS4ServiceComponentsConfig() {
       [CHECKOUT_FEATURE]: {
         cmsComponents: S4_SERVICE_CMS_COMPONENTS,
       },
+      [ORDER_FEATURE]: {
+        cmsComponents: [
+          'CancelOrderComponent',
+          'CancelOrderConfirmationComponent',
+          'ReturnOrderComponent',
+          'ReturnOrderConfirmationComponent',
+          'AccountOrderDetailsActionsComponent',
+          'AccountOrderDetailsItemsComponent',
+          'AccountOrderDetailsTotalsComponent',
+          'AccountOrderDetailsOverviewComponent',
+          'AccountOrderDetailsBillingComponent',
+          'AccountOrderDetailsGroupedItemsComponent',
+          'AccountOrderDetailsSimpleOverviewComponent',
+          'AccountOrderHistoryComponent',
+          'ReplenishmentDetailItemsComponent',
+          'AccountOrderDetailsReorderComponent',
+          'ReplenishmentDetailTotalsComponent',
+          'ReplenishmentDetailShippingComponent',
+          'ReplenishmentDetailActionsComponent',
+          'ReplenishmentDetailOrderHistoryComponent',
+          'AccountReplenishmentHistoryComponent',
+          'ReturnRequestOverviewComponent',
+          'ReturnRequestItemsComponent',
+          'ReturnRequestTotalsComponent',
+          'OrderReturnRequestListComponent',
+          'OrderConfirmationThankMessageComponent',
+          'OrderConfirmationItemsComponent',
+          'OrderConfirmationTotalsComponent',
+          'OrderConfirmationOverviewComponent',
+          'OrderConfirmationShippingComponent',
+          'OrderConfirmationBillingComponent',
+          'OrderConfirmationContinueButtonComponent',
+          'ReplenishmentConfirmationMessageComponent',
+          'ReplenishmentConfirmationOverviewComponent',
+          'ReplenishmentConfirmationItemsComponent',
+          'ReplenishmentConfirmationTotalsComponent',
+          'MyAccountViewOrderComponent',
+          'RescheduleServiceOrder',
+        ],
+      },
```

Comment:
Since we are making changes in the Order Detail page, we are specifying all the CMS components mentioned in the order library (`feature-libs\order\root\order-root.module.ts`) and adding our own CMS component `RescheduleServiceOrder` to the array here.